ote/src/cljs/ote/app/controller/login.cljs

```diff
@@ -0,0 +1,90 @@
+(ns ote.app.controller.login
```

Comment:
Next step is to create/register users as well. So consider renaming this controller to user. And we can add user creation functions to the same controller.